app/controllers/boards_controller.rb

```diff
@@ -0,0 +1,70 @@
+class BoardsController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:show]
```

Comment:
It is remanent code from a time where Devise was not fully configurated. It will be removed and changed on a future update altogether with other auth configuration.

Reply:
understood.